app/javascript/application.js

```diff
@@ -1,14 +1,15 @@
 // Configure your import map in config/importmap.rb. Read more: https://github.com/rails/importmap-rails
 import "@hotwired/turbo-rails";
 import "controllers";
+import "utilities/form";
```

Comment:
**question:** Do we need to add this here, by doing so it gets loaded onto every page

Reply:
Looks like it does not need to be here. Thanks for catching this. Updated in 8ee769258c40d65ed8637d4ffad500a93608af7f